CHANGELOG.md

```diff
@@ -140,6 +140,48 @@ The library has been expanded to include the v2 service.
 consider this a breaking change, as Bazel 5.x has been in maintenance mode for
 more than 6 months.
 
+**CMake Proto Libraries**: We no longer compile certain protos, unless the
+corresponding client library is enabled, via `-DGOOGLE_CLOUD_CPP_ENABLE=...`.
+
+This change reduces build times with `cmake` for customers who build with
+`cmake` and who are not using **all** of the client libraries listed below.
+
+The impacted libraries are:
+
+- `bigquery`
+- `bigtable`
+- `dialogflow_es`
+- `iam`
+- `logging`
+- `pubsub`
+- `speech`
+- `storage`
+- `texttospeech`
+- `trace`
+
+If you are dependent on any of these proto libraries **and** you are not
+compiling the corresponding client library, you will need to update your build
+scripts.
+
+For example, if you depend on `google_cloud_cpp_speech_protos`, (e.g. if you
+have been using this library to make calls to Cloud Speech using raw gRPC), add
+`-DGOOGLE_CLOUD_CPP_ENABLE=...,speech,...` to your CMake configure command. If
+you build with `vcpkg`, include `speech` in your install command.
+
+Note that `google_cloud_cpp_storage_protos` are associated with the
+`experimental-storage-grpc` feature, not the `storage` feature.
+
+We do not make changes that can break our customers lightly. We considered it
```

Comment:
Maybe move the paragraph earlier, maybe before "The impacted libraries".  And maybe consider something like:

> Compiling these protos when not strictly needed was a bug for some customers, and confusing to other customers.
> Any change in behavior, including fixing bugs, can be considered "breaking". By policy we don't consider bug fixes to be breaking changes. And we applied that policy in this case.

Skip the comment about the number of customers affected.

Reply:
> Maybe move the paragraph earlier, maybe before "The impacted libraries".

Done.

>  maybe consider something like:

I used your second paragraph, but I preferred my first paragraph.

> Skip the comment about the number of customers affected.

Done.